src/test/resources/sample2vid.json

```diff
@@ -0,0 +1,114 @@
+{
+        "contigs": {
+                "22": {
+                        "length": 51304566,
+                        "tiledb_column_offset": 0
+                }
+        },
+        "callset_mapping_file": "src/test/resources/sample2callsets.json",
+        "fields": {
+                "AD": {
+                        "vcf_field_class": ["FORMAT"],
+                        "type": "int"
+                },
+                "DP": {
+                        "vcf_field_class": ["FORMAT", "INFO"],
+                        "type": "int",
+			"VCF_field_combine_operation": "sum"
+                },
+                "GQ": {
+                        "vcf_field_class": ["FORMAT"],
+                        "type": "int"
+                },
+                "GT": {
+			"vcf_field_class": ["FORMAT"],
+                        "type": "int",
+                        "length": "P"
+                },
+                "PL": {
+                        "vcf_field_class": ["FORMAT"],
+                        "type": "int",
+                        "length": "G"
+                },
+		"AC": {
+                        "vcf_field_class": ["INFO"],
+                        "type": "int",
+			"length": "A",
+			"VCF_field_combine_operation": "median"
```

Comment:
@danking at first I specified it as "mean" or "median" just to get it working, and then I changed all the ones that were giving me errors. [Here](https://github.com/Intel-HLS/GenomicsDB/wiki/Importing-VCF-data-into-GenomicsDB#fields-information) is where I found the available combine operations.